train/train.py

```diff
@@ -48,27 +50,75 @@ def self_play(model_path, out_dir, config):
                    stderr=sys.stderr, stdout=sys.stdout, check=True)
 
 
-def train(model_path, data_dir, config):
-    pass  # TODO
+def train(model_path, _, config):
+    # TODO change
+    data_dir = Path(config["data_dir"])
+
+    def reverse_position(pos, winner):
+        return [-x for x in pos], -winner
+
+    positions = []
+    winners = []
+
+    logging.debug('a')
+
+    # Load all games made by the current model
+    for training_iteration_dir in data_dir.iterdir():
+        if str(model_id(model_path)) not in str(training_iteration_dir):

```

Comment:
we should probably call model_id(.) less, it's a slow function.
Also, i think just reading all the games from data_dir is better, rather than doing it by name